### PR TITLE
Use consistent naming for the remaining SGRs part two

### PR DESCRIPTION
**Pull request by olemarkus**
Continuation of #10179

### CODE REVIEW

File: pkg/model/firewall.go [hunk 1]
```diff
@@ -441,6 +441,6 @@ func generateName(o *awstasks.SecurityGroupRule) string {
 		src = fi.StringValue(o.SecurityGroup.Name)
 	}
 
-	return fmt.Sprintf("%s-%s-%s-%dto%d-%s", src, direction,
+	return fmt.Sprintf("from-%s-%s-%s-%dto%d-%s", src, direction,
```
Comment (rdrgmnzs) [inline pkg/model/firewall.go:444]: @olemarkus please correct me if I'm wrong, but won't changing the existing resource name for the terraform resources cause all resources to be re-created, effectively causing downtime while the changes are applied?

cc: @rifelpet

**Comment by olemarkus** [comment]
Yeah, I think so. I mentioned that concern and @rifelpet commented here:
https://github.com/kubernetes/kops/pull/10179#issuecomment-723444677

**Pull request merged**